content/millennium/dstu2/individuals/patient.md

```diff
@@ -71,11 +75,28 @@ Notes:
 - The `birthdate` parameter may be provided twice to indicate a date range, but must contain the inclusive prefixes 'le' and 'ge'
 - The `birthdate` parameter may be provided once with the following prefixes: 'ge', 'le', 'gt', 'lt', 'eq'
 
-### Response
+### Headers
+
+To successfully GET a Patient, the following headers must be provided. Authorization must be provided if accessing the closed endpoint.
+
+        Accept-Type: application/json+fhir
+        Authorization: <OAuth2 Bearer Token>
```

Comment:
Should these be indented 4 spaces instead of 8 to match the formatted block in Request?